src/MainWindow.vala

```diff
@@ -1217,7 +1217,6 @@ namespace Terminal {
         }
 
         private void on_get_text (Gtk.Clipboard board, string? intext) {
-            /* if unsafe paste alert is enabled, show dialog */
             if (Application.settings.get_boolean ("unsafe-paste-alert") && !unsafe_ignored ) {
 
```

Comment:
Extraneous white space

Reply:
@jeremypw what's extraneous here?

Reply:
Isn't the space on the end of the if clasule?